packages/blade/package.json

```diff
@@ -250,7 +250,7 @@
     "react-native": "0.72.3",
     "react-native-gesture-handler": "2.9.0",
     "react-native-reanimated": "3.4.2",
-    "react-native-svg": "12.3.0",
+    "react-native-svg": "15.1.0",
```

Comment:
Why?

Reply:
v12 types are not compatible with react18 types. 

+ we and most of our consumers are using react-native >v70, thus according to this [table](https://github.com/software-mansion/react-native-svg?tab=readme-ov-file#supported-react-native-versions) we should be using v15 of rnsvg.